helm_deploy/hmpps-incentives-ui/values.yaml

```diff
@@ -11,7 +11,7 @@ generic-service:
 
   ingress:
     enabled: true
-    contextColour: green
+    v1_2_enabled: true
```

Comment:
We could test this first in dev by updating `values-dev.yaml`, rather than here which will apply it to all envs?